pkg/pxc/app/binlogcollector/binlog-collector.go

```diff
@@ -201,7 +208,7 @@ func GetDeployment(cr *api.PerconaXtraDBCluster, initImage string) (appsv1.Deplo
 		},
 	}
 
-	if cr.CompareVersionWith("1.16.0") >= 0 {
+	if cr.CompareVersionWith("1.16.0") >= 0 && cr.CompareVersionWith("1.18.0") < 0 {
```

Comment:
Here, if we only have `cr.CompareVersionWith("1.18.0") < 0` are we covered? Maybe we can ditch the check for 1.16 since the check for 1.18 covers it, and version 1.15 and older will not be supported anymore.

Reply:
https://docs.percona.com/percona-operator-for-mysql/pxc/update.html#upgrade-the-operator-and-crd
We will support 1.18.0, 1.17.0 and 1.17.0